src/layouts/default.vue

```diff
@@ -106,6 +98,28 @@ export default {
       description: state => state.description,
       action:      state => state.action
     }),
+    defaultRoutes() {
+      return [
+        '/General',
+        '/K8s',
+        '/PortForwarding',
+        '/Images',
+        '/Troubleshooting'
+      ];
+    },
+    /**
+     * Inserts host preferences route at array index 1
+     */
+    modifiedRoutes() {
+      return [
+        ...this.defaultRoutes.slice(0, 1),
+        '/HostPreferences',
```

Comment:
Whatever we end up calling the tab that is currently called `Host/App`, this should match it.